diff --git a/lib/l10n/app_en.arb b/lib/l10n/app_en.arb
--- a/lib/l10n/app_en.arb
+++ b/lib/l10n/app_en.arb
@@ -1006,7 +1006,7 @@
     "defaultIcon": "Default",
     "oldIcon": "Old Icon",
     "switchAppIcon": "Switch App Icon",
-    "switchAppIconConfirm": "Switch to \"{label}\"? The app will restart. Continue?",
+    "switchAppIconConfirm": "Switch to \"{label}\"?",
     "@switchAppIconConfirm": {
         "placeholders": {
             "label": {
diff --git a/lib/l10n/app_zh.arb b/lib/l10n/app_zh.arb
--- a/lib/l10n/app_zh.arb
+++ b/lib/l10n/app_zh.arb
@@ -944,7 +944,7 @@
     "defaultIcon": "默认图标",
     "oldIcon": "旧版图标",
     "switchAppIcon": "切换应用图标",
-    "switchAppIconConfirm": "切换至「{label}」后应用将重启，是否继续？",
+    "switchAppIconConfirm": "是否切换至「{label}」？",
     "@switchAppIconConfirm": {
         "placeholders": {
             "label": {
