diff --git a/lib/pages/dev/dev_page.dart b/lib/pages/dev/dev_page.dart
--- a/lib/pages/dev/dev_page.dart
+++ b/lib/pages/dev/dev_page.dart
@@ -1,4 +1,3 @@
-import 'package:bugaoshan/pages/dev/ui/ui_tile.dart';
 import 'package:flutter/material.dart';
 
 import 'package:bugaoshan/injection/injector.dart';
@@ -137,8 +136,6 @@
           const Divider(),
           const AuthLogTile(),
           const Divider(),
-          const UiTile(),
-          const Divider(),
           const ChangelogTile(),
           const Divider(),
           ValueListenableBuilder<bool>(
